governator-core/src/main/java/com/netflix/governator/conditional/Conditional.java

```diff
@@ -0,0 +1,37 @@
+package com.netflix.governator.conditional;
+
+/**
+ * Contract for any conditional that may be applied to a conditional binding
+ * bound via {@link ConditionalBinder}.
+ * 
+ * Dependencies needed by a concrete conditional should be injected using 
+ * member injection.
+ * 
+ */
+public abstract class Conditional {
+    /**
+     * Evaluate whether the condition is true.  evaluate() is only called once at injector
+     * creation time.
+     * @return True if conditional is true otherwise false
+     */
+    public abstract boolean evaluate();
```

Comment:
I'm on the fence on what to call this.  Evaluate() seems more consistent with predicate terminology than matches().